### PR TITLE
RichCommand: Open File Location

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related to #11481 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Right-click a shortcut and see open file location works as expected

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Actions/FileSystem/OpenFileLocationAction.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+namespace Files.App.Actions
+{
+	internal class OpenFileLocationAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "OpenFileLocation".GetLocalizedResource();
+
+		public string Description => "TODO: Need to be described.";
```
Comment (yaira2) [inline src/Files.App/Actions/FileSystem/OpenFileLocationAction.cs:23]: Can you add a description?

File: src/Files.App/Helpers/ContextFlyoutItemHelper.cs [hunk 2]
```diff
@@ -257,14 +257,7 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					ShowInSearchPage = true,
 					ShowItem = itemsSelected && showOpenItemWith
 				},
-				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "OpenFileLocation".GetLocalizedResource(),
-					Glyph = "\uE8DA",
-					Command = commandsViewModel.OpenFileLocationCommand,
-					ShowItem = itemsSelected && selectedItems.All(i => i.IsShortcut),
-					ShowInSearchPage = true,
```
Comment (yaira2) [inline src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: Will this still show in the search page?

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Actions/FileSystem/OpenFileLocationAction.cs (hunk 1 above)
Comment (hecksmosis) [reply src/Files.App/Actions/FileSystem/OpenFileLocationAction.cs:23]: Ok

File: src/Files.App/Helpers/ContextFlyoutItemHelper.cs (hunk 2 above)
Comment (hecksmosis) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: I don't know, shortcuts do not appear in search results

Comment (yaira2) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: Search results can be from multiple locations so this allows users to open the location of the item.

Comment (hecksmosis) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: In stable the command that appears is open parent folder, not open file location

Comment (yaira2) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: Is that a separate action?

Comment (hecksmosis) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:266]: Yes

**Pull request merged**